### PR TITLE
Contributing.md

### PR DESCRIPTION
**Pull request by emcarru**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+- [Flynt Docs Issues](https://github.com/flyntwp/flynt-docs/issues)
+- [Flynt CLI Issues](https://github.com/flyntwp/flynt-cli/issues)
+- [Flynt Generator Issues](https://github.com/flyntwp/generator-flynt/issues)
+- [Flynt Docker Issues](https://github.com/flyntwp/docker-flynt-build/issues)
+- [ACF Field Composer Issues](https://github.com/flyntwp/acf-field-group-composer/issues)
+
+### Issues and Bugs
+Before you submit your issue please search GitHub issues first, maybe your question was already answered.
+
+If your issue appears to be a bug, and hasn't been reported, open a new issue. Help us to maximize the effort we can spend fixing issues and adding new features, by not reporting duplicate issues. Providing the following information will increase the chances of your issue being dealt with quickly:
```
Comment (dgrdl) [inline CONTRIBUTING.md]: "Help us maximize" without "to", right?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+- **Overview of the Issue** - if an error is being thrown a non-minified stack trace helps.
```
Comment (dgrdl) [inline CONTRIBUTING.md]: comma: "if an error is being thrown, a non-minified stack trace helps." also, should these begin with uppercase? (since they're technically sentences)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+- **Motivation for or Use Case** - explain why this is a bug for you.
```
Comment (dgrdl) [inline CONTRIBUTING.md]: "Motivation for or Use Case" this sounds slightly strange, would reword somehow

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+- **Major Changes** should be discussed first [in an issue](#issues-and-bugs) so that we can better coordinate our efforts, prevent duplication of work, and help you to craft the change so that it is successfully accepted into the project.
+- **Small Changes** can be crafted and submitted to the [relevant GitHub repository](#submission-guidelines) as a Pull Request.
+
+
+## Pull Requests
+Before you submit a pull request please adhere to the following guidelines:
+
+- Search the [relevant GitHub repository](#submission-guidelines) for an open or closed pull request that relates to your submission. You don't want to duplicate effort.
+
+- Make your changes in a new git branch. **If you are submitting a change for the [Flynt Start Theme](https://github.com/flyntwp/flynt-starter-theme), branch from the `develop` branch, not `master`.**
```
Comment (dgrdl) [inline CONTRIBUTING.md]: Start*er*

Comment (dgrdl) [reply CONTRIBUTING.md]: branch from or branch off?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+```
+git checkout -b myFixBranch master
+```
```
Comment (dgrdl) [inline CONTRIBUTING.md]: I would put the previous bolded information after this example

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+- Build and test your changes locally to ensure that everything works.
+
+- Push your branch to GitHub:
+
+```
+git push origin myFixBranch
+```
+
+- In GitHub, send a pull request to the `master` branch. **If you are submitting a pull request to the [Flynt Start Theme](https://github.com/flyntwp/flynt-starter-theme), send your pull request to the `develop` branch, not `master`.**
```
Comment (dgrdl) [inline CONTRIBUTING.md]: Start*er*

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+- If the pull request gets too out-dated, we may ask you to rebase and force push to update the pull request:
+
+```
+git rebase master -i
+git push origin myFixBranch -f
+```
+
+_WARNING: Squashing or reverting commits and force-pushing thereafter may remove GitHub comments on code that were previously made by you or others in your commits. Avoid any form of rebasing unless necessary._
```
Comment (dgrdl) [inline CONTRIBUTING.md]: unless absolutely necessary

File: CONTRIBUTING.md (hunk 4 above)
Comment (dgrdl) [inline CONTRIBUTING.md]: Is this actually correct? Can just anyone create a branch on our repos? Don't they have to fork it first and then do a PR from their repo to the main repo? Or am I missing something?

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+We follow the [AngularJS git commit guidelines](https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#-git-commit-guidelines) for all Flynt repositories. This leads to more readable and consistent messages that are easy to follow when looking through the project history. Please adhere to these guidelines.
+
+The commit message formatting can be added using a typical git workflow or through the use of the CLI wizard [Commitizen](https://github.com/commitizen/cz-cli).
+
+### PHP
+When writing PHP, follow the [PSR-2 coding style guide](http://www.php-fig.org/psr/psr-2/#overview).
+
+### JavaScript
+When writing JavaScript, follow the [Standard JS](https://standardjs.com/#the-rules) style guide.
```
Comment (dgrdl) [inline CONTRIBUTING.md]: with PHP it's coding style guide, here it's just style guide

**Review by dgrdl — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (emcarru) [reply CONTRIBUTING.md]: I'll remove the 'for'

File: CONTRIBUTING.md (hunk 4 above)
Comment (emcarru) [reply CONTRIBUTING.md]: Yeah you're right. That makes more sense, to be honest.

Comment (emcarru) [reply CONTRIBUTING.md]: I think 'from' is more universal

**Review by dgrdl — APPROVED** [review]
